subworkflows/local/shortread_preprocessing.nf

```diff
@@ -0,0 +1,188 @@
+/*
+ * SHORTREAD_PREPROCESSING: Preprocessing and QC for short reads
+ */
+
+include { FASTQC as FASTQC_RAW                                  } from '../../modules/nf-core/fastqc/main'
+include { FASTQC as FASTQC_TRIMMED                              } from '../../modules/nf-core/fastqc/main'
+include { FASTP                                                } from '../../modules/nf-core/fastp/main'
+include { ADAPTERREMOVAL as ADAPTERREMOVAL_PE                   } from '../../modules/nf-core/adapterremoval/main'
+include { ADAPTERREMOVAL as ADAPTERREMOVAL_SE                   } from '../../modules/nf-core/adapterremoval/main'
+include { BOWTIE2_REMOVAL_BUILD as BOWTIE2_HOST_REMOVAL_BUILD   } from '../../modules/local/bowtie2_removal_build'
+include { BOWTIE2_REMOVAL_ALIGN as BOWTIE2_HOST_REMOVAL_ALIGN   } from '../../modules/local/bowtie2_removal_align'
+include { BOWTIE2_REMOVAL_BUILD as BOWTIE2_PHIX_REMOVAL_BUILD   } from '../../modules/local/bowtie2_removal_build'
+include { BOWTIE2_REMOVAL_ALIGN as BOWTIE2_PHIX_REMOVAL_ALIGN   } from '../../modules/local/bowtie2_removal_align'
+include { CAT_FASTQ                                            } from '../../modules/nf-core/cat/fastq/main'
+include { SEQTK_MERGEPE                                        } from '../../modules/nf-core/seqtk/mergepe/main'
+include { BBMAP_BBNORM                                         } from '../..//modules/nf-core/bbmap/bbnorm/main'
```

Comment:
Please align :)

If you use VSCode, you should install the Nextflow extension which now includes a formatter you can activate via the command palette and  should do that for you :)

Reply:
Will do. thanks for suggestion.